main/py-requests/APKBUILD

```diff
@@ -3,14 +3,14 @@
 pkgname=py-requests
 _pkgname=requests
 pkgver=2.18.1
-pkgrel=0
+pkgrel=1
 pkgdesc="A HTTP request library for Python"
 url="http://www.python-requests.org/"
 arch="noarch"
 license="ASL 2.0"
 # Requirements for tests are not available in main
 options="!check"
-depends=""
+depends="py-chardet py-idna py-certifi py-urllib3"
```

Comment:
py2-*/py3-* packages should not depend on py-*. See https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python#Multiversion_package_without_executables.